frontend/app/views/event/thankyou.scala.html

```diff
@@ -45,7 +45,7 @@ <h2 class="page-section__headline">Find more events</h2>
             </div>
         </section>
 
-        @fragments.analytics.eventConversion(event, order)
+        @fragments.analytics.eventConversion()
```

Comment:
I guess we're just now recording that we sold an event, rather than which event that was - but there is a separate system, ie EventBrite, for getting that information!